roles/prereqs/meta/main.yml

```diff
@@ -1,4 +1,3 @@
 ---
-
 dependencies:
   - prereqs/libselinuxpython
```

Comment:
My thought was that we would make other roles depend only on `prereqs`, and then `prereqs` would install all prerequisites.  You're using a more granular approach (roles are depending on, for example, `prereqs/libsemanagepython` directly), which is fine, but that means we should probably just get rid of this file (and the `meta` directory).